src/sonic-config-engine/setup.py

```diff
@@ -85,6 +85,7 @@
     install_requires = dependencies,
     data_files = [
         ('/usr/share/sonic/templates', glob.glob('data/*')),
+        ('sonic/dns', ['data/dns.j2']),
```

Comment:
Why not place the template in the template folder above?

Why not using the absolute path?

Reply:
For absolute path like '/usr/share/sonic/templates', the target folder is:
/usr/local/lib/python2.7/dist-packages/usr/share/sonic/templates/l2switch.j2
/usr/local/lib/python3.7/dist-packages/usr/share/sonic/templates/l2switch.j2
For relative path like 'sonic/dns', the target folder is: /usr/local/sonic/dns

Reply:
And it's difficult to locate path like "/usr/local/lib/python3.7/dist-packages/usr/share/sonic/templates/"